packages/eas-cli/src/vcs/index.ts

```diff
@@ -7,7 +7,9 @@ const NO_VCS_WARNING = `Using EAS CLI without version control system is not reco
 
 function resolveVcsClient(): Client {
   if (process.env.EAS_NO_VCS) {
-    log.warn(NO_VCS_WARNING);
+    if (process.env.NODE_ENV !== 'test') {
```

Comment:
shouldn't we just mock console.log/warn in tests?

Reply:
Actually, those are mocked but this code executes before `jest` manages to mock the functions :|